modules/benchmark/R/calc.benchmark.R

```diff
@@ -18,28 +18,31 @@ calc.benchmark <- function(settings, con){ #settings file is output from start.b
   
   # Query BRR for benchmark ids
   
-  bm.id <- settings.bm$run$id #maybe something like that?
+  bm.id <- settings$benchmark$id #maybe something like that?
   
   # For each bm.id
   #  Query database for: input path, variables, site, start/end dates, metrics, formats
   
-  input.path <- db.query.file.path(input.id,host_name,con)
-  f_v <- db.query.format.vars(input.id,con)
-  format_table <- f_v[[1]]
-  vars_names_units <- f_v[[2]]
+  data.path <- query.file.path(bm.id,settings$run$host$name,con)
+  format <- query.format.vars(bm.id,con)  
   
   
   # Need to do something here to narrow down the variables to be used
   # rows <- which(vars_names_units$bety_name %in% calc.vars)
   # vars_names_units <- vars_names_units[rows,]
   
-  site  <- db.query.site(input.id, con)
+  site  <- query.site(settings$run$site$id, con)
```

Comment:
No, it's because the settings won't nessisarily contain model run info when benchmarking